app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
     package="com.example.sharingang">
 
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+    <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.WAKE_LOCK"/>
```

Comment:
Why do you need that permission?

Reply:
This was when I was using the initial version, I needed this for the login to be possible, otherwise I was getting weird errors in the console. Fortunately, with the new implementation it works well without wake lock, I forgot to remove it, but will do.